src/app/store/machine/slice.ts

```diff
@@ -1508,6 +1508,42 @@ const machineSlice = createSlice({
     setZoneError: statusHandlers.setZone.error,
     setZoneStart: statusHandlers.setZone.start,
     setZoneSuccess: statusHandlers.setZone.success,
+    softOff: {
+      prepare: (params: BaseMachineActionParams) => {
+        let actionParams;
+        if ("filter" in params) {
+          const { filter, system_id, ...extra } = params;
+          actionParams = {
+            action: NodeActions.OFF,
+            filter,
+            system_id,
+            extra: { ...extra, stop_mode: "soft" },
+          };
+        } else {
+          const { system_id, ...extra } = params;
+          actionParams = {
+            action: NodeActions.OFF,
+            extra: { ...extra, stop_mode: "soft" },
+            system_id,
+          };
+        }
+        return {
+          meta: {
+            model: MachineMeta.MODEL,
+            method: "soft_power_off",
+          },
+          payload: {
+            params: actionParams,
+          },
+        };
+      },
+      reducer: () => {
+        // No state changes need to be handled for this action.
+      },
+    },
```

Comment:
This should be covered by tests. You can add them in `src/app/store/machine/reducers.test.ts`, you could start by adding a new test case with:

`expect(reducers(initialState, actions.softOff())).toEqual(`